package.json

```diff
@@ -72,8 +72,8 @@
     "fs-extra": "^7.0.1",
     "glob": "^7.1.2",
     "humanize": "0.0.9",
-    "ioslib": "^1.7.7",
-    "liveview": "^1.4.5",
+    "ioslib": "^1.7.9",
+    "liveview": "^1.5.0",
```

Comment:
Could you back the liveview change out (and maybe change to `~1.4.5`), it intended to only land on the 8_1_X branch to align with the node-titanium-sdk changes that landed there